org.apache.aries.events.api/src/main/java/org/apache/aries/events/api/SubscribeRequest.java

```diff
@@ -35,15 +38,36 @@ private SubscribeRequest(String topic, Consumer<Received> callback) {
         this.callback = callback;
     }
 
+    /**
+     * Build a subscription request for the given topic and {@code Consumer} callback.
+     *
+     * @param topic to consume from
+     * @param callback to be invoked for each message consumed
+     * @return a new subscription request
+     */
     public static SubscribeRequest to(String topic, Consumer<Received> callback) {
         return new SubscribeRequest(topic, callback);
     }
-    
+
+    /**
+     * Set the {@code Position} position to start consuming from.
+     *
+     * @param position in the topic to start consuming from
+     * @return the updated subscribe request
+     */
     public SubscribeRequest startAt(Position position) {
         this.position = position;
         return this;
     }
-    
+
+    /**
+     * Set the earliest or latest position to start consuming from
+     * when the position is {@code null} or not valid. By default,
+     * seek is set to {@link Seek#earliest}.
```

Comment:
The default is latest

Reply:
Thanks for spotting this. Updated the PR.